src/main/java/com/ays/admin_user/controller/AdminUserRegisterApplicationController.java

```diff
@@ -12,4 +23,31 @@
 @RequestMapping("/api/v1/admin")
 @RequiredArgsConstructor
 public class AdminUserRegisterApplicationController {
```

Comment:
Bu sınıfı dışarıdan çağırmadığımız için `package-private` yapabiliriz

Reply:
Duzelttim.